tests/test_stationsBlockedViews.py

```diff
@@ -0,0 +1,138 @@
+        assert response.status_code == 200
+
+    def test_get_stations_blocked_response(self, admin, factory, station_blocked):
+        request = factory.get('/api/stations/blocked')
+        request.headers = {'Authorization': f'Bearer {admin.user.username}'}
+
+        response = stations_blocked(request)
+        assert json.loads(response.content) == [
+            {
+                'id': station_blocked.pk,
+                'name': station_blocked.name
+            }
+        ]
+
+    def test_post_stations_blocked_user_status(self, factory, user, station_working):
+        body = json.dumps({'id': station_working.pk})
+        request = factory.post('/api/stations/blocked', content_type = 'application/json', data = body)
+        headers = {'Authorization': f'Bearer {user.user.username}'}
+        headers.update(request.headers)
+        request.headers = headers
+
+        response = stations_blocked(request)
+        assert response.status_code == 403
+
+    def test_post_stations_blocked_tech_status(self, factory, tech, station_working):
+        body = json.dumps({'id': station_working.pk})
+        request = factory.post('/api/stations/blocked', content_type = 'application/json', data = body)
+        headers = {'Authorization': f'Bearer {tech.user.username}'}
+        headers.update(request.headers)
+        request.headers = headers
+
+        response = stations_blocked(request)
+        assert response.status_code == 403
+
+    def test_post_stations_blocked_admin_status(self, factory, admin, station_working):
+        body = json.dumps({'id': station_working.pk})
+        request = factory.post('/api/stations/blocked', content_type = 'application/json', data = body)
+        headers = {'Authorization': f'Bearer {admin.user.username}'}
+        headers.update(request.headers)
+        request.headers = headers
+
+        response = stations_blocked(request)
+        assert response.status_code == 201
+
+    def test_post_stations_blocked_on_blocked(self, factory, admin, station_blocked):
+        body = json.dumps({'id': station_blocked.pk})
+        request = factory.post('/api/stations/blocked', content_type = 'application/json', data = body)
+        headers = {'Authorization': f'Bearer {admin.user.username}'}
+        headers.update(request.headers)
+        request.headers = headers
+
+        response = stations_blocked(request)
+        assert response.status_code == 422
+
+    def test_post_stations_blocked_response(self, factory, admin, station_working):
+        body = json.dumps({'id': station_working.pk})
+        request = factory.post('/api/stations/blocked', content_type = 'application/json', data = body)
+        headers = {'Authorization': f'Bearer {admin.user.username}'}
+        headers.update(request.headers)
+        request.headers = headers
+
+        response = stations_blocked(request)
+        assert response.status_code == 201
+        assert json.loads(response.content) == {
```

Comment:
I don't think 2 asserts in one tests are correct, it should be two separate tests

Reply:
done